src/fbw/src/ThrottleAxisMapping.cpp

```diff
@@ -77,6 +77,22 @@ bool ThrottleAxisMapping::loadFromLocalVariables() {
   return true;
 }
 
+bool ThrottleAxisMapping::setDefaults() {
```

Comment:
suggestion: `applyDefaults()` might describe the action a bit better as we don't set/change defaults but apply the to the config. Very subjective though so proceed as you wish :)